### DISCUSSION TITLE
Ignore bot comments on real time alerts on a PR<title>

### DISCUSSION THREAD

**Discussion by brainpicker** [community/community#110968] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

We have added an integration for a bot that comments on our PR's and gives us feedback. It does a lot of comments on it and I get tons of notifications now. 
Would love to add some ignore list or something or option to ignore bots.

**Comment by Illia65** [comment]
Response#1

**Comment by meganoceros** [comment]
agreed. we have github action and bots adding comments and we'd like to exclude those
